plugin_runner/plugin_runner.py

```diff
@@ -413,27 +440,17 @@ def refresh_event_type_map() -> None:
                 log.warning(f"Unknown RESPONDS_TO type: {type(responds_to)}")
 
 
-def load_plugins(specified_plugin_paths: list[str] | None = None) -> None:
```

Comment:
The whole "specified_plugin_paths" is how the cli command `canvas run-plugins` works. You can list specific plugin paths to run your local plugin runner with. Can we keep that functionality?

Reply:
we sure can but I couldn't find the usage

Reply:
ah I see because it doesn't call it directly it calls `run_server(...)`

Reply:
that proves `mypy` is broken in that repository, should error out on an argument change like that!